Dockerfile

```diff
@@ -8,5 +8,8 @@ RUN make install-requirements && make VERIFY=true all
 ############# gardener-extension-runtime-gvisor
 FROM alpine:3.11.3 AS gardener-extension-runtime-gvisor
 
+COPY charts /charts
 COPY --from=builder /go/bin/gardener-extension-runtime-gvisor /gardener-extension-runtime-gvisor
+COPY --from=builder /go/src/github.com/gardener/gardener-extension-runtime-gvisor/bin/containerd-shim-runsc-v1.linux-amd64 /var/content/containerd-shim-runsc-v1.linux-amd64
+COPY --from=builder /go/src/github.com/gardener/gardener-extension-runtime-gvisor/bin/runsc /var/content/runsc
```

Comment:
I dont find the runsc and shim binaries - have they been forgotten / am I missing something?

Reply:
They were forgotten to be pushed